sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -77,6 +77,13 @@ describe(
       systemUsersService = app.get(SystemUsersService);
       // Create a Ministry user to b used, for instance, for audit.
       sharedMinistryUser = await db.user.save(createFakeUser());
+      // Insert fake email contact to send ministry email.
+      await db.notificationMessage.update(
+        {
+          id: NotificationMessageType.MinistryNotificationDisbursementBlocked,
+        },
+        { emailContacts: ["dummy@some.domain"] },
+      );
```

Comment:
Asking for my understanding. Was this required?

Reply:
Shashank asked the same question, and I replied in his comment: [here](https://github.com/bcgov/SIMS/pull/4010#discussion_r1862674639).